capture/Makefile

```diff
@@ -41,9 +46,31 @@ else
 	CFLAGS += -O3
 endif
 
+ifeq ($(PLATFORM), x86)
+	CFLAGS += -m32
+endif
+
 ifeq ($(PLATFORM), x64)
 	CFLAGS += -m64
-	CFLAGS += -lrt
+endif
+
+ifeq ($(PLATFORM), armv5)
+#	CC      = arm-none-linux-gnueabi-g++
+	CFLAGS += -march=armv5
+endif
+
+ifeq ($(PLATFORM), armv6)
+#	CC      = arm-linux-gnueabi-g++
+	CFLAGS += -march=armv6
+endif
+
+ifeq ($(PLATFORM), armv7)
+#	CC      = arm-linux-gnueabihf-g++
+	CFLAGS += -march=armv7 -mcpu=cortex-m3 -mthumb
+endif
+
+ifeq ($(PLATFORM), armv8)
+#	CC = aarch64-linux-gnu-g++
 endif
```

Comment:
What's going on with the commented-out `CC` lines?

Reply:
So, those are based (slightly loosely) on what's in the original demo makefile here: https://github.com/seeing-things/zwo/blob/master/1.16.3/linux_sdk/demo/Makefile#L53-L85

In principle they would be necessary if you were, for example, cross-compiling the `capture` program *on* your Intel box *for* your ARM box with your gcc cross-compiler for ARM. Because in that situation, the main/non-prefixed gcc is the one that builds binaries targeted for your Intel box. And the prefixed version is the one that builds binaries targeted for the ARM box.

When building *for* the ARM box *on* the ARM box itself, you don't actually really need the toolchain prefix, because the main/non-prefixed gcc in that situation would be the one targeting the ARM box itself.

(Fun exercise: I bet that on any of your AMD64 boxes, you can run regular gcc via just `gcc` but *also* via `x86_64-pc-linux-gnu-gcc` or similar. But because the host and target arch are the same, you normally just use `gcc` to build stuff.)

So... yeah, these prefixes are only really needed for cross-compiling... but somebody might want them... I dunno... in any event, I halfheartedly left them in, in commented-out form, as a sort of "whatever" compromise.